Language_Parser/actionVerbs.py

```diff
@@ -257,13 +258,37 @@ def go(info):
 
 
 def look(info):
-    """"""
-    pass
+    """
+    Action function prints the long description of the Player's current room.
+    """
+    print(info["Player"].getLocation().getLongDescription())
 
 
 def eat(info):
-    """"""
-    pass
+    """
+    Action function prints response to the Player attempting to eat various
+    Items or Room features.
+    """
+    if len(info["Items"]) == 0:
+        print("You're thinking about eating something, but what?")
+        return
+    # Get target name that the player wants to eat from input
+    eatTarget = info["Items"][0]
+    # Look for item with target name in player inventory and current room
+    inventoryList = info["Player"].getInventory()
+    roomItemList = info["Player"].getLocation().getItems()
+    allItems = inventoryList + roomItemList
+    for item in allItems:
+        if eatTarget == item.getName().lower():
```

Comment:
You need to make "eatTarget" lowercase as well to handle items such as silverBell, catFood, canOpener, and tinCan

if eatTarget.lower() == item.getName().lower():